web-app/src/utilities/AxiosInstance.js

```diff
@@ -18,36 +18,6 @@ instance.interceptors.response.use(function(response) {
     return response;
   }
 
-  if (response.config.url.includes(`${baseURL}/holidays`)) {
-    const holidays = [...response.data];
-    for (const index in holidays) {
-      holidays[index].employee = {
-        employeeId: holidays[index].employeeId,
-        forename: 'Test',
-        surname: 'User',
-        email: 'test@user.com',
-        totalHolidays: 33,
-        startDate: [2014, 1, 1],
-        countryId: 1,
-        employeeRoleId: 2,
-        employeeStatusId: 2,
-      };
-
-      holidays[index].start = new moment(
-        holidays[index].startDate,
-        'YYYY-MM-DD',
-      );
```

Comment:
Is there a reason you've removed the creation of moment creation here?

Reply:
on getting holiday events in dashboard container, i'm removing properties not required, so it was just as handy to sort the moment dates out there too.

Reply:
@paulmatchett But by removing it here you're removing it from anywhere else that makes the holiday calls.